typing/includecore.ml

```diff
@@ -70,6 +70,22 @@ type value_mismatch =
 
 exception Dont_match of value_mismatch
 
+let value_descriptions_consistency env vd1 vd2 =
```

Comment:
Do you explain somewhere what "consistency" means, so that we can understand the intended different between the `foo` and the `foo_consistency` functions?

Reply:
I have added a tentative definition of consistency


> (** The functions [value_descriptions_consistency] and
    [type_declarations_consistency] check if two declaration are consistent.
    Declarations are consistent when there exists an environment such that the
    first declaration is a subtype of the second one.

>    Notably, if a type declaration [td1] is consistent with [td2] then a type
    expression [te] which is well-formed with the [td2] declaration in scope
    is still well-formed with the [td1] declaration: [E, td2 |- te] => [E, td1
    |- te]. *)

with the idea that the property which is needed is the second one.